source/backtraceApi.ts

```diff
@@ -34,8 +44,8 @@ export class BacktraceApi {
           rej(e);
         };
       });
-    } catch (err) {
-      return BacktraceResult.OnError(report, err);
+    } catch (err: unknown) {
```

Comment:
do we need to case the error object to  unknown to cast it once again to Error? Wouldn't it be better if we cast it only once on the return statement?